Dockerfiles/kubectl-gadget.Dockerfile.dockerignore

```diff
@@ -4,6 +4,7 @@
 !cmd
 !pkg
 !internal/thirdparty
+!internal/deployinfo
```

Comment:
Why not everything in `internal` right away to avoid this breaking again in the future once we add another package?

Reply:
No good reason, I should have done that indeed.